data/korath/korath weapons.txt

```diff
@@ -577,6 +737,138 @@ effect "slicer impact"
 	"random velocity" .1
 	"velocity scale" -.002
 
+
+outfit "Korath Digger Turret"
+	category "Turrets"
+	cost 509000
+	thumbnail "outfit/korath digger turret"
+	"mass" 14
+	"outfit space" -16
+	"weapon capacity" -16
+	"asteroid scan power" 15
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/digger"
+			"frame rate" 20
+		"hardpoint sprite" "hardpoint/digger turret"
+		sound "korath digger"
+		"hit effect" "korath digger hit"
+		"inaccuracy" 1.5
+		"turret turn" 4.6
+		"velocity" 480
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 2.9
+		"firing heat" 4.9
+		"shield damage" 0.8
+		"hull damage" 1.3
+		"minable damage" 5.2
+		"disruption damage" .08
+	description `When the Korath Exiles needed a cheap mining laser for their large vessels, they re-engineered this ancient but robust design. The mild shield disruption effect makes it somewhat useful in combat.`
```

Comment:
In a pinch a mining laser would probably work, and the disruption damage is fairly nifty, but I wouldn't trust this to be my main weapon, and once the exiles salvage the meretri disruptor turrets I can see these being relegated to be purely for mining.
```suggestion
	description `When the Korath Exiles needed a cheap mining laser for their large vessels, they re-engineered this ancient but robust design. The mild shield disruption effect makes it somewhat useful in combat if required.`
```

Reply:
Yeah, in testing I've noticed a single Digger is incapable of overcoming the shield and hull regen of a tiny systems core.   Definitely nothing more than a "I can't fit anything else in this hardpoint" kind of weapon.